common/src/x86emitter/x86emitter.cpp

```diff
@@ -214,8 +202,6 @@ const char* xRegisterBase::GetName()
 		case 4: return x86_regnames_gpr32[ Id ];
 #ifdef __x86_64__
```

Comment:
Is it correct that for 32bit 8 is not a valid operand size? Or did you rather wanted to use the gpr64 for both 32 bit and 64 bit?